### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by Carloslarrea9** [Launch-X-Latam/MisionFrontEnd#142] [Q&A]
hola buenas noches compañero.

acabo de terminar mi practica se las comparto para sabe si esta bien y si me falto en agregar algo 

este es mi repositorio por si gustan entrar 
https://github.com/Carloslarrea9/ABOGABOT.git

**Comment by doguedogue** [comment]
la veo vacia compañero 🚀

**Comment by VictorFalconi** [comment]
Acao de entrar y no hay nada xd Has commiteado tus cambios?
